src/YaccConstructor/Program.fs

```diff
@@ -44,6 +44,32 @@ let log (e:System.Exception) msg =
     "\nMessage:\n  " + msg
     |> printfn "%s"
 
+open Argu
+
+type CLIArguments =
+    | [<Inherit; AltCommandLine("-f","-frontend")>] F of feName:string option
+    | [<AltCommandLine("-af","-availableFrontends")>] Af
```

Comment:
[```Unique``` ] (http://fsprojects.github.io/Argu/reference/argu-arguattributes-uniqueattribute.html) attribute should be specified for some args.